lib/postinstall/update-package.js

```diff
@@ -11,6 +11,7 @@ const changes = {
   scripts: {
     lint: `eslint '**/*.js'`,
     postlint: 'npm-template-check',
+    'template-copy': 'npm-template-copy --force',
```

Comment:
do we want to add this script to everything? are we re-applying templates enough that it's necessary?